steps/20/README.md

```diff
@@ -62,13 +63,16 @@ Additionally, we set the formatting option `showMeasure` to `false`. This hides
       items="{invoice>/Invoices}" >
       <items>
          <ObjectListItem
+            core:require="{
+               Currency: 'sap/ui/model/type/Currency'
```

Comment:
Can you also provide some explanation in the description what this code means and why it is needed so people understand and can adapt from here to any other use cases?

Reply:
Done